Modules/Filtering/QuadEdgeMeshFiltering/test/itkDiscreteGaussianCurvatureQuadEdgeMeshFilterTest.cxx

```diff
@@ -58,19 +58,20 @@ itkDiscreteGaussianCurvatureQuadEdgeMeshFilterTest(int argc, char * argv[])
 
   MeshType::Pointer mesh = reader->GetOutput();
 
-  auto gaussian_curvature = CurvatureFilterType::New();
-  gaussian_curvature->SetInput(mesh);
-  gaussian_curvature->Update();
+  auto gaussianCurvatureFilter = CurvatureFilterType::New();
 
-  MeshType::Pointer output = gaussian_curvature->GetOutput();
+  gaussianCurvatureFilter->SetInput(mesh);
+  gaussianCurvatureFilter->Update();
+
+  MeshType::Pointer output = gaussianCurvatureFilter->GetOutput();
 
   using WriterType = itk::MeshFileWriter<MeshType>;
   auto writer = WriterType::New();
   writer->SetInput(output);
   writer->SetFileName("gaussian_curvature.vtk");
   writer->Update();
 
-  // ** PRINT **
-  std::cout << gaussian_curvature;
+
+  std::cout << gaussianCurvatureFilter;
```

Comment:
Should there be "Test finished" message here?

Reply:
Done here: https://github.com/InsightSoftwareConsortium/ITK/pull/4286/files#diff-bf9342a65397b9400ab9a8de8e41bd5f0b03dd695630d86cb7e15dc5d53905edR76